app/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
   <application
     android:name=".HedvigApplication"
     android:allowBackup="true"
-    android:enableOnBackInvokedCallback="false"
+    android:enableOnBackInvokedCallback="true"
```

Comment:
That's to do with the back gesture, right? Could you briefly explain here what problem was there with the sheets?

Reply:
Yes it's this https://developer.android.com/guide/navigation/custom-back/predictive-back-gesture

So back some releases ago, material3 sheets were somehow unable to be dismissed when you did a back gesture when this option was on.

This was the issue https://issuetracker.google.com/issues/271303558
And this https://android-review.googlesource.com/c/platform/frameworks/support/+/2885946 commit seems to have been the fix, which was released in `material3:1.2.0-beta02`, which we have been using for quite some time now.

Reply:
Great, thanks